docs/src/languages/ruby.md

```diff
@@ -71,6 +71,18 @@ To switch to `ruby-lsp`, add the following to your `settings.json`:
   "languages": {
     "Ruby": {
       "language_servers": ["ruby-lsp", "!solargraph", "!rubocop", "..."]
+    },
+    // Enable herb and ruby-lsp for *.html.erb files
+    "HTML+ERB": {
+      "language_servers": ["herb", "ruby-lsp", "..."]
```

Comment:
Following the note above, I think we could leave it as `["ruby-lsp", "..."]` but I want to show that we explicitly enable both `herb` and `ruby-lsp`.